api_helpers.py

```diff
@@ -73,19 +79,37 @@ def get_files(in_str, record_dict=None):
             yield 1
 
```

Comment:
Well, this question caught me! I need to closely review the logic again. 

Basically, I wrote this generator function to avoid rate limiting imposed on some CZO servers.
This function parses/downloads a czo resource, yields to function Upload_HS_File() for file creation. Thanks to the slow performance of our HS REST API (just kidding...), the elapsed time of HS file creation is just enough to avoid the rate limiting that would block moving onto the next CZO resource...So basically I use hs file creation time to replace the need of time.sleep()

Reply:
probably wouldn't help you with the rate limiting. Also probably not hurting... was just curious